packages/components/src/navigator/stories/index.story.tsx

```diff
@@ -244,54 +238,43 @@ export const NestedNavigator: StoryObj< typeof NavigatorProvider > = {
 		initialPath: '/child2/grandchild',
 		children: (
 			<>
-				<NavigatorScreen path="/">
-					<Card>
-						<CardBody>
-							<NavigatorButton variant="secondary" path="/child1">
-								Go to first child.
-							</NavigatorButton>
-							<NavigatorButton variant="secondary" path="/child2">
-								Go to second child.
-							</NavigatorButton>
-						</CardBody>
-					</Card>
-				</NavigatorScreen>
-				<NavigatorScreen path="/child1">
-					<Card>
-						<CardBody>
-							This is the first child
-							<NavigatorBackButton variant="secondary">
-								Go back to parent
-							</NavigatorBackButton>
-						</CardBody>
-					</Card>
-				</NavigatorScreen>
-				<NavigatorScreen path="/child2">
-					<Card>
-						<CardBody>
-							This is the second child
-							<NavigatorBackButton variant="secondary">
-								Go back to parent
-							</NavigatorBackButton>
-							<NavigatorButton
-								variant="secondary"
-								path="/child2/grandchild"
-							>
-								Go to grand child.
-							</NavigatorButton>
-						</CardBody>
-					</Card>
-				</NavigatorScreen>
-				<NavigatorScreen path="/child2/grandchild">
-					<Card>
-						<CardBody>
-							This is the grand child
-							<NavigatorBackButton variant="secondary">
-								Go back to parent
-							</NavigatorBackButton>
-						</CardBody>
-					</Card>
-				</NavigatorScreen>
+				<StyledNavigatorScreen path="/">
```

Comment:
I thought it would be a good practice to set each screen to be full height — do you think it's actually a bad practice?

Reply:
Honest question — why is it a good practice? Does it ensure that the screens animate properly or something? I'm not familiar enough with the component to know why it should be that tall. As a consumer I just see some relatively short content and a lot of whitespace 🙈

Reply:
So, here's a full explanation:

It's not important for the screens to be full height or to have `height: 100%`.

What matters is that the `NavigatorProvider` (soon to be renamed `Navigator`) wrapper has enough height to host the tallest screen. Otherwise, when transitioning between screens (and especially with the changes proposed in https://github.com/WordPress/gutenberg/pull/64777), the content may "jump" (or get cut off) unexpectedly.

Reply:
Thanks for investigating, that makes sense! Maybe something to document as well.